internal/k8s/entity.go

```diff
@@ -89,7 +89,7 @@ func (e K8sEntity) ToObjectReference() v1.ObjectReference {
 		Kind:       kind,
 		APIVersion: apiVersion,
 		Name:       meta.GetName(),
-		Namespace:  meta.GetNamespace(),
```

Comment:
hmmm...i don't think this is accurate?

empty namespace != "default", it's equal to the kube context namespace...

Reply:
augh, so the comment is wrong, i meant it normalized to _the default_ rather than "default", but I foolishly thought that `k8s.DefaultNamespace` was the default for your kubecontext whoops. If we don't have a good way to a. figure out and b. access the default ns for your context, then ugh. (and unless there's a really easy answer, I suspect this isn't worth worrying about right now.)

Reply:
ya, the DefaultNamespace was a mistake and is something i've been meaning to remove :eyes: